src/UI/Buyer/src/app/shared/components/no-results/no-results.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input, EventEmitter, Output } from '@angular/core';
+import { Meta } from '@angular/platform-browser';
+
+@Component({
+  selector: 'shared-no-results',
+  templateUrl: './no-results.component.html',
+  styleUrls: ['./no-results.component.scss'],
+})
+export class NoResultsComponent {
+  @Input() message: string;
+  @Input() actionText: string;
+  @Output() action = new EventEmitter<void>();
+
+  constructor(private meta: Meta) {
+    this.meta.addTag({ name: 'robots', content: 'noindex' });
```

Comment:
So I added this kind of late to winmark but can we also have it remove this take `onDestroy` otherwise the robots meta tag will be there after they've browsed a page that isn't found which means seo won't take effect on pages it should